docs/modules/SDDs/pages/0022-managed-services-in-cluster.adoc

```diff
@@ -0,0 +1,111 @@
+= SDD 0022 - Managed Services in Cluster
+
+:sdd_author:    Tobias Brunner
+:sdd_owner:
+:sdd_reviewers:
+:sdd_date:      2020-06-19
+:sdd_status:    draft
+include::partial$meta-info-table.adoc[]
+
+[NOTE]
+.Summary
+====
+Besides the system services Project Syn also provides additional services to the user of the platform. This SDD describes how some kind of managed services on Kubernetes with Project Syn can be achieved. It is considered to be the initial version of this concept.
+====
+
+== Motivation
+
+While the current state of Project Syn focuses on delivering system services to Kubernetes clusters, it should also take application services into consideration. There needs to be a way to install, configure and maintain application services and make them available to the user of the platform.
```

Comment:
The term _application service_ is used throughout the document. For me, this is not a well defined thing and needs definition within this document.

Reply:
I was missing a better name and therefore used it. I could also imagine to call this "middleware", but this sounds a bit too enterprisey to me. Do you have any other idea? Of course a definition / glossary would also help.

Reply:
I do not much care about the term used as long as it is well defined. If a general definition is not given, as it is the case for _application service_, one must be made within the scope of the discussion.

I do not have any suggestion if and how to change that term. I assume that once a definition is written down, we might come up with something more suitable.

Reply:
I'll add a Terminology section then.